Dockerfile

```diff
@@ -1,32 +1,15 @@
-FROM denoland/deno:ubuntu-1.37.1
+FROM golang:1.21
```

Comment:
I'd expect README to tell how I run this program locally, or how do I run tests =)